pkg/cleanup/object_cleanup.go

```diff
@@ -0,0 +1,51 @@
+package cleanup
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+type SpecCleaner interface {
```

Comment:
Either `Cleaner` or just `Interface`. Or move it to the package where it is consumed. Or to the root package?